bin/gen_peptides.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+
+import sys
+import argparse
+import pandas as pd
+
+from Bio.SeqIO.FastaIO import SimpleFastaParser
+from Fred2.Core import Allele, Peptide, Protein,generate_peptides_from_proteins
+
+
+parser = argparse.ArgumentParser("Generating peptides from protein sequences.")
+parser.add_argument('-i', '--input', metavar='FILE', type=argparse.FileType('r'), help = 'FASTA filename containing proteins.')
+parser.add_argument('-o', '--output', metavar='FILE', type=argparse.FileType('w'), help='Output file: peptides.')
+parser.add_argument('-min', '--min_length', metavar='N', type=int, help='Min. length of peptides that will be generated.')
```

Comment:
I think we can use the full words here in the help message: `Minimal`and `Maximum`